<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoanStackedPage</class>
 <widget class="QWidget" name="LoanStackedPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="FinishButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>570</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>归还</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ProlongButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>570</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>续期</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Search">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="LoanTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>621</width>
     <height>411</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>用户账号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书籍编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>起始日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>截止日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>借阅序号</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ReserveButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>570</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>预约成功</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
